docs/user/openstack/README.md

```diff
@@ -501,6 +501,14 @@ spec:
           availabilityZone: <optional_openstack_availability_zone>
 ```
 
+[provider-spec-definition]: https://github.com/openshift/cluster-api-provider-openstack/blob/155384b859c5b2fb5b7f11c9111d3f8e4f3066bd/pkg/apis/openstackproviderconfig/v1alpha1/types.go#L31
+
+#### Defining a New MachineSet That Uses Multiple Networks
+
+To attach multiple networks to a machineSet, the `primarySubnet` value in the `providerSpec` must be set to the [machines subnet](./customization.md#cluster-scoped-properties). If this value is not set correctly, the nodes will be unable to join the cluster.
```

Comment:
here you use "machineSet" (lower case), but in L447 you use "MachineSet" is that correct?

Reply:
You mention `primarySubnet` in `providerSpec` in ./customization.md#cluster-scoped-properties. 
and it sounds as if you are saying that the user needs to set `primarySubnet` somewhere, but if i am reading this correctly you mean that the user needs to set the `machineSubnet` which in turn will set the `primarySubent` in the `providerspec` of the generated machineSet... right?  anyways. I think its a bit hard to follow.

Reply:
I tried cleaning up the sentence. Unfortunately, I cant clean up the UX. I agree that its unclear and I dont like it. Lets make an epic for that if we want to change it.